docs/conf.py

```diff
@@ -35,6 +35,7 @@
     "sphinx.ext.autodoc",
     "sphinx.ext.napoleon",
     "sphinx_click.ext",
+    "sphinx_automodapi.automodapi",
```

Comment:
Has this dependency been captured anywhere?

Reply:
it's in the requirements.txt ?

Reply:
Ah I didn't see it in there.